src/components/DSRDepositScreens/DSRDepositCheckProxy.js

```diff
@@ -1,147 +1,49 @@
 import React from 'react';
-import {
-  Box,
-  Text,
-  Card,
-  Button,
-  Grid,
-  Tooltip
-} from '@makerdao/ui-components-core';
+import { Box, Text } from '@makerdao/ui-components-core';
 
 import lang from 'languages';
 import ScreenFooter from './ScreenFooter';
-import useBlockHeight from 'hooks/useBlockHeight';
 import useProxy from 'hooks/useProxy';
-import useTokenAllowance from 'hooks/useTokenAllowance';
 
-import { ReactComponent as Checkmark } from 'images/checkmark.svg';
-import TooltipContents from 'components/TooltipContents';
-
-const SuccessButton = () => {
-  return (
-    <Button variant="primary-outline" width="13.0rem" mt="xs" disabled>
-      <Checkmark />
-    </Button>
-  );
-};
+import ProxyAllowanceCheck from '../ProxyAllowanceCheck';
 
 const DSRDepositCheckProxy = ({ dispatch, onClose }) => {
-  const blockHeight = useBlockHeight(0);
-
+  const symbol = 'MDAI';
+  const labels = {
+    setup_text: lang.dsr_deposit.setup_proxy_text
+  };
   const {
+    proxyLoading,
+    hasProxy,
     proxyAddress,
+    proxyDeployed,
     setupProxy,
-    proxyLoading,
     startingBlockHeight,
-    proxyDeployed,
-    proxyErrors,
-    hasProxy
+    proxyErrors
   } = useProxy();
 
-  const {
-    hasAllowance,
-    setAllowance,
-    allowanceLoading: isSettingAllowance
-  } = useTokenAllowance('DAI');
+  console.log(
+    '%%1PALuseProxy vals: proxyAddress,proxyLoading,startingBlockHeight,proxyDeployed, proxyErrors',
+    proxyAddress,
+    // setupProxy,
+    proxyLoading,
+    startingBlockHeight,
+    proxyDeployed,
+    proxyErrors
+  );
```

Comment:
For some reason, these state variables aren't updating in this component, even after the state is changed in the `useProxy` hook. However in the child component, the same variables (and log) are being updated as expected.